plugin-server/tests/worker/ingestion/person-state-batch.test.ts

```diff
@@ -2235,7 +2235,7 @@ describe('PersonState.processEvent()', () => {
             expect(person.is_identified).toEqual(true)
 
             const result = await hub.db.postgres.query(
-                PostgresUse.COMMON_WRITE,
+                PostgresUse.PERSONS_READ,
```

Comment:
question: in practice this shouldn't change anything in the test environment, but shouldn't we be pointing to PERSONS_WRITE?